card/mod/05_standard/spec/set/all/history_spec.rb

```diff
@@ -147,16 +151,20 @@
 
       context 'when updated' do
         it 'adds act for left card' do
-          @card.update_attributes subcards: {"+right"=>{content: "New content", db_content: "New Content"}}
-          expect(Card::Act.count).to eq(act_start_cnt+2)
+          @card.update_attributes subcards: {
+            '+right' => {
+              content: 'New content', db_content: 'New Content'
+            }
+          }
+          expect(Card::Act.count).to eq(act_start_cnt + 2)
           expect(act.card).to eq(@card)
         end
         it 'adds action for subcard' do
-          @card.update_attributes subcards: {"+right"=>{content: "New Content"}}
+          @card.update_attributes subcards: {'+right'=>{content: 'New Content'}}
```

Comment:
Surrounding space missing for operator `=>`.<br>Space inside { missing.<br>Space inside } missing.